src/Tribe/Views/V2/Hooks.php

```diff
@@ -299,13 +301,25 @@ public function filter_map_canonical_query_args( $map, $url, $rewrite ) {
 	 *
 	 * @return array $classes
 	 */
-	public function filter_body_class( $classes ) {
-		$classes = $this->container->make( Theme_Compatibility::class )->filter_add_body_classes( $classes );
+	public function filter_body_classes( $classes ) {
 		$classes = $this->container->make( Template_Bootstrap::class )->filter_add_body_classes( $classes );
 
 		return $classes;
 	}
 
+	public function add_body_classes() {
```

Comment:
:no_entry_sign: Missing doc comment for function add_body_classes()

<sub>hash: 9b1861e2ca063d2bef6384dbf15c6024</sub>